src/contentmanager.cpp

```diff
@@ -585,6 +587,18 @@ void ContentManager::updateRemoteLibrary(const QString& content) {
     });
 }
 
+void ContentManager::updateLanguages(const QString& content) {
+    m_remoteLibrary = kiwix::Library();
```

Comment:
This re initialize the`m_remoteLibrary`. Why we need this ?
If we update categories after languages, we will lost the known languages.

Reply:
This is what happens when I forget things to remove while editing commits in a rebase.
Thanks and fixed.